### PR TITLE
Build

### PR DESCRIPTION
**Pull request by malo-bo**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
Make NuGet restore work and build MAUI sample and NuGet without glitches

### :arrow_heading_down: What is the current behavior?
It did not restore not build

### :new: What is the new behavior (if this is a feature change)?
It now works

### :boom: Does this PR introduce a breaking change?
Not to my knowledge

### :bug: Recommendations for testing
N/A

### :memo: Links to relevant issues/docs
N/A

### :thinking: Checklist before submitting

- [ ] All projects build
- [ ] Follows style guide lines 
- [ ] Relevant documentation was updated
- [ ] Rebased onto current develop

### CODE REVIEW

File: Lottie.iOS/Lottie.iOS.csproj [hunk 1]
```diff
@@ -10,6 +9,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsBindingProject>true</IsBindingProject>
     <Version>2.6.0</Version>
```
Comment (JesperTroiborgLaursen) [inline Lottie.iOS/Lottie.iOS.csproj:11]: Should Version and ReleaseVersion be aligned? :)

File: Samples/Forms/Example.Forms.UWP/Example.Forms.UWP.csproj [hunk 2]
```diff
@@ -20,29 +20,6 @@
     <PackageCertificateKeyFile>Example.Forms.UWP_TemporaryKey.pfx</PackageCertificateKeyFile>
     <ReleaseVersion>2.5</ReleaseVersion>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
```
Comment (JesperTroiborgLaursen) [inline Samples/Forms/Example.Forms.UWP/Example.Forms.UWP.csproj:23]: Do we need these configs for people not on ARM cpus? :)

File: Samples/Native/Example.macOS/Example.macOS.csproj [hunk 3]
```diff
@@ -21,7 +21,7 @@
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <EnableCodeSigning>false</EnableCodeSigning>
+    <EnableCodeSigning>False</EnableCodeSigning>
```
Comment (JesperTroiborgLaursen) [inline Samples/Native/Example.macOS/Example.macOS.csproj:24]:
```suggestion
    <EnableCodeSigning>false</EnableCodeSigning>
```

File: Samples/Native/Example.macOS/Example.macOS.csproj [hunk 4]
```diff
@@ -43,7 +43,7 @@
     </DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <EnableCodeSigning>false</EnableCodeSigning>
+    <EnableCodeSigning>False</EnableCodeSigning>
```
Comment (JesperTroiborgLaursen) [inline Samples/Native/Example.macOS/Example.macOS.csproj:46]:
```suggestion
    <EnableCodeSigning>false</EnableCodeSigning>
```

**Review by JesperTroiborgLaursen — APPROVED** [review]
Minor questions, but looks good :)

File: Samples/Forms/Example.Forms.UWP/Example.Forms.UWP.csproj (hunk 2 above)
Comment (malo-bo) [reply Samples/Forms/Example.Forms.UWP/Example.Forms.UWP.csproj:23]: There is no more x86. It's x64 by now 😉

File: Samples/Native/Example.macOS/Example.macOS.csproj (hunk 3 above)
Comment (malo-bo) [reply Samples/Native/Example.macOS/Example.macOS.csproj:24]: Visual studio for Mac - not me

File: Lottie.iOS/Lottie.iOS.csproj (hunk 1 above)
Comment (malo-bo) [reply Lottie.iOS/Lottie.iOS.csproj:11]: Don't think so 😉

**Pull request merged**